src/main.js

```diff
@@ -545,6 +549,16 @@ class MiniGraphCard extends LitElement {
     `;
   }
 
+  renderLabelsSecondary() {
+    if (!this.config.show.labels_secondary) return;
+    return html`
+      <div class="graph__labels flex" style="right: 0">
```

Comment:
Let's move the inline style to `style.js`, also need to get rid of the right-margin.
Class can be "graph__labels --secondary" or something so we can target it.

Reply:
Yeah this is the sort of tidying I have yet to do. I'm assuming "--secondary" will only have "right: 0; margin-right: 0px" in it.

Reply:
Done